bin/migration-scripts/README.md

```diff
@@ -1,35 +1,47 @@
 # Astronomer upgrade guide
 
+If you are already on an LTS version and you are seeking a patch version update or to just reconfigure the software, please skip to the bottom section "patch version updates"
 
-## Schedule a change window
+## Astronomer versioning: moving to the long-term-support (LTS) software versioning model
+
+We have customers running Astronomer versions 0.7 through 0.16, and we currently support all of them. The wide variety of configurations and versions present in our customer's environments and the accelerating new minor version release cycle has presented challenges with regard to training support enginnering, backporting fixes, documenting change notes, and with upgrades. In July 2020, Astronomer decided to start following the long-term-support (LTS) model for enterprise-ready Astronomer versions. This will look like some of our minor versions receive continuous patching while the others do not. This allows our developers to focus on building new features and limiting the number of versions where they need to backport patches. In the near future, we will be only publicly publishing the LTS versions. The release cadence is not yet decided but it will be somewhere in between 1 month and 6 months, depending on how many LTS versions we plan to support at the same time.
```

Comment:
```suggestion
We have customers running Astronomer versions 0.7 through 0.16, and we currently support all of them. The wide variety of configurations and versions present in our customer's environments and the accelerating new minor version release cycle has presented challenges with regard to training support engineering, backporting fixes, documenting change notes, and with upgrades. In July 2020, Astronomer decided to start following the long-term-support (LTS) model for enterprise-ready Astronomer versions. This will look like some of our minor versions receive continuous patching while the others do not. This allows our developers to focus on building new features and limiting the number of versions where they need to backport patches. In the near future, we will be only publicly publishing the LTS versions. The release cadence is not yet decided but it will be somewhere in between 1 month and 6 months, depending on how many LTS versions we plan to support at the same time.
```